tests/test_tags.py

```diff
@@ -271,18 +285,63 @@ def test_mac_platforms(self):
             "macosx_10_5_intel",
             "macosx_10_5_fat64",
             "macosx_10_5_fat32",
+            "macosx_10_5_universal2",
             "macosx_10_5_universal",
             "macosx_10_4_x86_64",
             "macosx_10_4_intel",
             "macosx_10_4_fat64",
             "macosx_10_4_fat32",
+            "macosx_10_4_universal2",
             "macosx_10_4_universal",
         ]
 
-        assert len(list(tags.mac_platforms((10, 17), "x86_64"))) == 14 * 5
+        assert len(list(tags.mac_platforms((10, 17), "x86_64"))) == 14 * 6
 
         assert not list(tags.mac_platforms((10, 0), "x86_64"))
 
+    def _test_macos_11(self, major, minor):
+        platforms = list(tags.mac_platforms((major, minor), "x86_64"))
+        assert "macosx_11_0_arm64" not in platforms
+        assert "macosx_11_0_x86_64" in platforms
+        assert "macosx_11_3_x86_64" not in platforms
+        assert "macosx_11_0_universal" in platforms
+        assert "macosx_11_0_universal2" in platforms
+        # Mac OS "10.16" is the version number that binaries compiled against an old
+        # (pre 11.0) SDK will see.   It can also be enabled explicitly for a process
+        # with the environment variable SYSTEM_VERSION_COMPAT=1
+        assert "macosx_10_16_x86_64" in platforms
```

Comment:
The point of the 10.16 thing is to provide a workaround for binary compatibility.   It's applied to executables that were built against an old (pre-11.0) SDK.    But since arm was introduced in 11.0, it is impossible for a binary to be compiled for arm using an old SDK.   So this issue only effects x86.